src/components/tooltip/TrackPower.js

```diff
@@ -136,6 +125,45 @@ class TooltipTrackPowerImpl extends React.PureComponent<Props> {
     );
   }
 
+  renderForPreviewSelection(
+    previewSelection
+  ): React.ChildrenArray<React.Element<typeof TooltipDetail> | null> | null {
+    if (!previewSelection.hasSelection) {
```

Comment:
I think you could change this test to also return null when selectionStart == selectionEnd and simplify the rest of this function. I don't think showing "Energy consumed in the current selection: 0 Wh" has any value. It just looks bogus.